static/js/clusterImgFive.js

```diff
@@ -0,0 +1,49 @@
+const maskBtn 					= document.getElementById('clipMask');
+const guideBtn 					= document.getElementById('guideLine');
+const refreshBtn 				= document.getElementsByTagName('button')[0];
+const clustersCollection 		= document.getElementsByClassName('cluster');
+const clusterImagesCollection	= document.getElementsByClassName('cluster--images');
+const clusters 					= [].slice.call(clustersCollection);
+const clusterImages 			= [].slice.call(clusterImagesCollection);
+
+function init(){
+	addListeners();
+	positionImages();
+}
+
+function addListeners(){
+	refreshBtn.addEventListener('click', function(){
```

Comment:
Can be `refreshBtn.addEventListener('click', positionImages);`

Reply:
Fixed